src/utils/domainMetadata.ts

```diff
@@ -170,17 +170,6 @@ export const DomainMetadataConfig: EnvironmentMetadataConfigType = {
       blockExplorerUrl: "https://etherscan.io",
       renderName: "Ethereum",
     },
-    2: {
```

Comment:
@tcar121293 I think this will then impact showing historic data on UI?

Reply:
yes, forgot about that, will revert the change